site/_docs/reference.md

```diff
@@ -2290,7 +2290,32 @@ Note:
 #### Declaring Objects For Types Defined In Schema
 After an object type is defined and installed in the schema, you can use it to declare objects in any SQL block. For example, you can use the object type to specify the datatype of an attribute, column, variable, bind variable, record field, table element, formal parameter, or function result. At run time, instances of the object type are created; that is, objects of that type are instantiated. Each object can hold different values.
```

Comment:
Why this part is under `JSON Functions`, i think we should move it below the `DDL Extensions` part as a top level.

Reply:
@danny0405 Updated, kindly check if it is the right place?